packages/webcontainer-api-mock/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "webcontainer-api-mock",
+  "version": "1.0.0",
+  "description": "A mock for the WebContainer API",
+  "type": "module",
+  "bugs": "https://github.com/stackblitz/tutorialkit/issues",
+  "homepage": "https://github.com/stackblitz/tutorialkit",
+  "license": "MIT",
+  "exports": {
+    ".": "./src/index.ts"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b"
+  },
+  "devDependencies": {
+    "@webcontainer/api": "1.2.0",
+    "typescript": "^5.4.5",
+    "vitest": "^1.6.0"
+  }
+}
```

Comment:
I've been thinking that this might be useful to others that write tests for code that uses the WebContainer API and so maybe we should publish it as a result?

Or do you think it's best to have it named `test-utils` and postpone to later the idea of publishing it?

Reply:
Good point :+1: Let's go with `test-utils` then.